### PR TITLE
[Merged by Bors] - chore: adaptations for nightly-2024-12-05

### PR DESCRIPTION
**Pull request by jcommelin**

### CODE REVIEW

File: Mathlib/Tactic/ToLevel.lean [hunk 1]
```diff
@@ -17,31 +17,8 @@ override the ones from Lean 4 core.
...
-  toLevel : Level
-  /-- The universe itself. This is only here to avoid the "unused universe parameter" error. -/
-  univ : Type u := Sort u
-export ToLevel (toLevel)
 attribute [pp_with_univ] toLevel
 
-instance : ToLevel.{0} where
-  toLevel := .zero
-
 universe u v
```
Comment (grunweg) [inline Mathlib/Tactic/ToLevel.lean:22]: This line can also be removed, right?

**Review by grunweg — COMMENTED** [review]
Thanks, looks good to me. I found one more line to remove :-)
maintainer merge

File: Mathlib/Tactic/ToLevel.lean (hunk 1 above)
Comment (kim-em) [inline Mathlib/Tactic/ToLevel.lean:22]:
```suggestion
```

**Comment by kim-em** [comment]
bors merge

**Pull request closed without merging**